content/api-references/funding/transfers.md

```diff
@@ -56,13 +56,14 @@ Transfers allow you to transfer money/balance into your end customers' account (
 
 ### ENUM.TransferStatus
 
-| Attribute  | Description                                   |
-| ---------- | --------------------------------------------- |
-| `QUEUED`   | Transfer is in queue to be processed          |
-| `PENDING`  | Transfer is pending processing                |
-| `REJECTED` | Transfer is rejected                          |
-| `APPROVED` | Transfer is approved                          |
-| `COMPLETE` | Transfer is completed. This is a final state. |
+| Attribute           | Description                                       |
+| ------------------- | ---------------------------------------------     |
+| `QUEUED`            | Transfer is in queue to be processed              |
+| `PENDING`           | Transfer is pending processing                    |
+| `SENT_TO_CLEARING`  | Transfer is being processed by the clearing firm. |
+| `REJECTED`          | Transfer is rejected                              |
+| `APPROVED`          | Transfer is approved                              |
+| `COMPLETE`          | Transfer is completed. This is a final state.     |
```

Comment:
```suggestion
| Attribute                           | Description                                                              |
| ------------------------ | ----------------------------------------------- |
| `QUEUED`                       | Transfer is in queue to be processed                    |
| `APPROVAL_PENDING` | Transfer is pending approval                                 |
| `PENDING`                      | Transfer is pending processing                             |
| `SENT_TO_CLEARING`  | Transfer is being processed by the clearing firm |
| `REJECTED`                    | Transfer is rejected                                                 |
| `CANCELED`                   | Client initiated transfer cancellation                     |
| `APPROVED`                   | Transfer is approved                                              |
| `COMPLETE`                   | Transfer is completed                                            |
| `RETURNED`                   | The bank issued an ACH return for the transfer  |
```

Reply:
@sfritz thanks for opening this PR to update the transfer statuses! I added some other values that were missing from our docs. If you could go ahead and accept the suggestion I will approve and merge the PR.

Reply:
@cruales thanks for taking a look, I added your changes. It'd be great if you could also review these similar PRs and add any missing enum values to them:

https://github.com/alpacahq/bkdocs/pull/101
https://github.com/alpacahq/bkdocs/pull/96